opencti-platform/opencti-front/src/private/components/settings/decay/DecayRuleCreation.tsx

```diff
@@ -0,0 +1,353 @@
+import React, { FunctionComponent } from 'react';
+import { graphql, useMutation } from 'react-relay';
+import { Field, FieldArray, Form, Formik, FormikConfig } from 'formik';
+import Drawer, { DrawerVariant } from '@components/common/drawer/Drawer';
+import Button from '@mui/material/Button';
+import * as Yup from 'yup';
+import * as R from 'ramda';
+import { RecordSourceSelectorProxy } from 'relay-runtime';
+import makeStyles from '@mui/styles/makeStyles';
+import { AddOutlined, Delete } from '@mui/icons-material';
+import IconButton from '@mui/material/IconButton';
+import Typography from '@mui/material/Typography';
+import Tooltip from '@mui/material/Tooltip';
+import Box from '@mui/material/Box';
+import { InformationOutline } from 'mdi-material-ui';
+import ListItemText from '@mui/material/ListItemText';
+import AutocompleteField from '../../../../components/AutocompleteField';
+import { useFormatter } from '../../../../components/i18n';
+import ItemIcon from '../../../../components/ItemIcon';
+import MarkdownField from '../../../../components/MarkdownField';
+import TextField from '../../../../components/TextField';
+import type { Theme } from '../../../../components/Theme';
+import SwitchField from '../../../../components/SwitchField';
+import { fieldSpacingContainerStyle } from '../../../../utils/field';
+import { insertNode } from '../../../../utils/store';
+import useAuth from '../../../../utils/hooks/useAuth';
+import { handleErrorInForm } from '../../../../relay/environment';
+import { DecayRulesLinesPaginationQuery$variables } from './__generated__/DecayRulesLinesPaginationQuery.graphql';
+
+const useStyles = makeStyles<Theme>((theme) => ({
```

Comment:
`makeStyles` is deprecated. Try to put style directly in JSX using `style={{...}}` for html elements or `sx={{...}}` for MUI elements